manager/integration/tests/test_basic.py

```diff
@@ -4621,3 +4621,30 @@ def test_backup_volume_restore_with_access_mode(core_api, # NOQA
                          fromBackup=b.url)
     volume_sp_access_mode = client.by_id_volume(volume_name_sp_access_mode)
     assert volume_sp_access_mode.accessMode == overridden_restored_access_mode
+
+
+@pytest.mark.skip(reason="TODO")  # NOQA
+def test_delete_backup_during_restoring_volume():  # NOQA
+    """
+    Test delete backup during restoring volume
+
+    Context:
+
+    The volume status should be failed if the backup was deleted during
```

Comment:
The restore status of the volume engine? Or any status to check?

Reply:
Modified.
Check that volume robustness is `faulted`